tests/Naming/ExpectedNameResolver/InflectorSingularResolverTest.php

```diff
@@ -38,6 +38,8 @@ public static function provideData(): Iterator
         yield ['staticCallsToNews', 'staticCallToNew'];
         yield ['newsToMethodCalls', 'newToMethodCall'];
         yield ['hasFilters', 'hasFilter'];
+        yield ['parametersAcceptor', 'parametersAcceptor'];
+        yield ['parametersAcceptorWithPhpDoc', 'parametersAcceptorWithPhpDoc'];
```

Comment:
This should be generic for any value of "<some>s<another>" :+1:

Reply:
counter examples are a few lines above

Reply:
@TomasVotruba I agree that the test-cases
```
        yield ['staticCallsToNews', 'staticCallToNew'];
        yield ['newsToMethodCalls', 'newToMethodCall'];
```

do not sound like proper english.

Reply:
I have adjusted the test-cases to only strip the last "s". I think this makes total sense and the pre-existing test-cases are not worth keeping up.

we can't know wheter the code is referencing `news` in the sense of the news in the media, or whether "new"-objects are meant.